es/src/main/java/io/apiman/distro/es/Bootstrapper.java

```diff
@@ -51,23 +48,10 @@
     public void contextInitialized(ServletContextEvent sce) {
         DistroESConfig config = new DistroESConfig();
 
-        File esHome = getDataDir();
-        if (esHome == null) {
-            System.out.println("--------------------------------------------------------------");
-            System.err.println("No apiman-es data directory found.  Embedded ES *not* started.");
-            System.out.println("--------------------------------------------------------------");
-            return;
-        }
-        if (!esHome.exists()) {
-            esHome.mkdirs();
-        }
-
         System.out.println("------------------------------------------------------------");
         System.out.println("Starting apiman-es.");
         System.out.println("   HTTP Ports:      " + config.getHttpPortRange());
-        System.out.println("   Transport Ports: " + config.getTransportPortRange());
         System.out.println("   Bind Host:       " + config.getBindHost());
-        System.out.println("   ES Data Dir:     " + esHome);
```

Comment:
Actually, let's add this back in and allow persistent data between quickstart reboots!